invenio_records_lom/ui/records/records.py

```diff
@@ -128,24 +128,22 @@ def record_export(
         }
     )
     exported_record = serializer.serialize_object(record.to_dict())
-    return render_template(
-        "invenio_records_lom/records/export.html",
-        export_format=exporter.get("name", export_format),
-        exported_record=exported_record,
-        record=LOMUIJSONSerializer().serialize_object_to_dict(record.to_dict()),
-        permissions=record.has_permissions_to(["update_draft"]),
-        is_preview=is_preview,
-        is_draft=record._record.is_draft,
-    )
+    content_type = exporter.get("content-type", export_format)
+    filename = exporter.get("filename", export_format).format(id=pid_value)
+    headers = {
+        "Content-Type": content_type,
+        "Content-Disposition": f"attachment; filename={filename}",
+    }
+    return (exported_record, 200, headers)
```

Comment:
Does this change belong to the migration from setup.py to setup.cfg? It seams that it belongs to the export feature?

Reply:
Belongs to the export-feature update.
Removed.